tests/utils/test_databricks_utils.py

```diff
@@ -102,21 +102,28 @@ def test_databricks_single_slash_in_uri_scheme_throws(get_config):
         databricks_utils.get_databricks_host_creds("databricks:/profile:path")
```

Comment:
@dbczumar - Are there unit tests checking the precedence order of various configs? 
@victorsun123 Would be good to add Model Serving config there as well